flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkSchemaUtil.java

```diff
@@ -52,6 +59,8 @@
  */
 public class FlinkSchemaUtil {
 
+  public static final String flinkPrefix = "flink.";
```

Comment:
static variable name should be all upper cases.

also I am not sure this is the right class to host this constant. I am wondering if we should introduce a `FlinkTableProperties` to host this one.